### ISSUE TITLE
Captain backpack spawning with 2 new items upon joining request.

### ISSUE THREAD

**Issue by Killette2** [d3athrow/vgstation13#9098]
I'd like two new items to be in the captains backpack upon joining. When a captain spawns or joins the station, their backpack comes with ID changing materials and an energy gun. Standard issue things, but what if we added two more things?
1. A pinpointer. Sure there's one in his office, but that may get nabbed by many of the pesky things (janiborgs, greyshirts, tators, vox) and since we usually don't have a captain or we have a latejoin one at the lower hours, an extra pinpointer won't do any harm and may assist in disarming the nuke in a nuke ops round for an extra climatic attempt to disarm the nuke before it's too late.
2. A codebreaker. THIS SHOULD ONLY SPAWN ON THE CAPTAIN HIMSELF, NOT IN HIS OFFICE. There may be situations where everything's gone to shit and the best option is to nuke the station instead of calling the shuttle (xenos actually winning, blob winning, shouldn't be done during malf because you are trying to prevent that from happening), however if this is done admins should judge if the nuke was an appropriate action or not to help weed out shit captains from good ones. This can also be done to disarm the nuclear bomb when nuke ops try to bomb the station, and give traitors a very useful item if they manage to kidnap or assassinate the captain to threaten the station with. Surely nuking the station is dying a glorious death, and probably killing your target too!

If the later item seems like a horrifying idea, it doesn't really have to be included as it's not as important as a spare pinpointer, or a week trial period where captains get codebreakers would surely make a fun time.

**Comment by liltiptop** [comment]
1 seems okay, it's true that lots of captains late-join. 2 seems like a major change, the current rule on the station nuke is that if you can guess the code, then you can nuke the station, even as a non antag. The main issue that I see with the second one is that someone can take the nuke code, blow it up, and get away free. If you were to do the second one, the captain would need to know the code only to himself, like the nuke op leader's message saying "the code is: 12345." I like both ideas, to be honest family, it gives the captain more stuff to have.

> This can also be done to disarm the nuclear bomb when nuke ops try to bomb the station

Nuke ops get their OWN nuke, so it's useless for that.

**Comment by Killette2** [comment]
Well yes nuke ops do get their own nuke, but they still need the disk. If the HoS or someone else of importance has a spare pinpointer when the captain is down or didn't guard the pinpointer or disk, the HoS or HoP could try to get an idea of where the nuke was last placed and try to decode it, or unanchor and mass drive it if the fluke ops were bad enough to leave the disk behind.

This would make the nuke timer starting be not the end of the round in most cases, but more the most climactic part of it to see whether or not it can be stopped. I mentioned nothing about the station's bomb during a nuke ops round, but I'm pretty sure the code breaker works on any nuke.

And yes, nuking the station would be punishable if you did it for lel reasons, it's just a worse non-antag bombing. Or better depending on how you look at it, at least there's no repair needed to be done. I would say non-antag nuking would be on par with maliciously releasing a singulo or non-antag bombing, and logged.

I considered the captain only knowing the code, but interrogation may not be the most effective when it considers if the station is nuked or not. I would say a paper, but that could be viewed by ghost, so the codebreaker item seems like the better way to do it I'd guess. Maybe security would be alerted if a codebreaker was ever stripped off the captain, such as when security GPS's are stripped from security?

**Comment by Boxe** [comment]
i am against the second but the first is nice

also someone porting those TG condom headsets that give ear protection against flashbangs mit b cool

**Comment by NigglyWiggly** [comment]
Fucking what
No why the fuck would anyone ever be given a codebreaker in regular gameplay, jesus Killette.

**Comment by Senaat1** [comment]
Killette the codebreaker is such a bad idea holy shit :100: :-1:  this is not :ok_hand:

**Comment by clusterfack** [comment]
Codebreaker is obviously out of the question but a second pinpointer is viable.

**Comment by Shadowmech88** [comment]
I agree with the pinpointer.

The captain shouldn't get a codebreaker. Detonating a nuke is a big thing. It kills everyone on the station.
If the captain really, REALLY thinks the situation calls for a nuclear detonation, he can fax Centcomm or message them via the comms console and ask them for the nuke code.

**Comment by Killette2** [comment]
Well maybe if more admins answered/even responded to the syndie console request instead of being afkmins we could ask the syndicate for cool stuff like codebreakers instead of requesting them to be in game naturally :^)

**Comment by NigglyWiggly** [comment]
Maybe if players didn't use the syndie console once in a blue moon just to ask for a 'pulse rifle and nuke code pls i want to win' admins would be more willing to help out :^)

**Comment by Killette2** [comment]
Bull shieet, I can't even get them to say anything back half the time. One round as a clown two traitors fought to the death, but the loser used an explosive implant so after the fight we rushed in and looted them, then the RD wanted the e-sword from me. As the clown, I decided the best course of action would be to emag the console and tell the syndicate that I'm pawning their gear and asked how much each piece is worth. No reply
